src/views/Give/RecipientModal.tsx

```diff
@@ -36,30 +36,31 @@ type RecipientModalProps = {
   project?: Project;
 };
 
+const DECIMAL_PLACES = 2;
```

Comment:
not a showstopper, but I am curious on why you're thinking 2 decimals here. vs. the 4 decimals shown in most other places? I'm sure you have a good reason, just looking for your thought process.

Reply:
Given the space available in the modal window, 2 dp seemed to fit better.

I think there does need to be some guidance on the usage of DP though, as it's inconsistent:
- stake page: 4 dp
- zap: 2 or 4 dp
- give: 2 dp in general (unless required to be a specific number, e.g. deposit amount)